src/Microsoft.DotNet.Build.Tasks/project.json

```diff
@@ -1,8 +1,27 @@
 {
   "dependencies": {
-    "NuProj": "0.10.4-beta-gf7fc34e7d8"
+    "NuProj": "0.10.4-beta-gf7fc34e7d8",
+    "Microsoft.Build.Framework": "0.1.0-preview-00005",
+    "Microsoft.Build.Tasks.Core": "0.1.0-preview-00005",
+    "Microsoft.Build.Utilities.Core": "0.1.0-preview-00005",
+    "Microsoft.CSharp": "4.0.0",
+    "System.Linq": "4.0.0",
+    "System.Xml.XDocument": "4.0.0",
+    "System.IO.Compression": "4.0.0",
+    "System.IO.Compression.ZipFile": "4.0.0",
+    "System.Xml.ReaderWriter": "4.0.10",
+    "System.Xml.XPath.XmlDocument": "4.0.0",
+    "System.Reflection": "4.0.10",
+    "System.Reflection.Metadata": "1.1.0-alpha-*",
+    "Newtonsoft.Json": "7.0.1"
```

Comment:
Good catch! I forgot to remove package.config contents, I just pushed that change as well. Basically we are using project.json now instead for that.